src/appkit.cs

```diff
@@ -18712,6 +18712,18 @@ interface NSToolbarItem : NSCopying {
 		bool Autovalidates { get; set; }
 	}
 
+	[Mac (10,5)]
+	[BaseType (typeof(NSToolbarItem))]
+	interface NSToolbarItemGroup
+	{
+		[Export ("initWithItemIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string itemIdentifier);
```

Comment:
Whenever there's a designated initializer like this, i.e. that does not accept `null` for references, then it makes me think the default `init` might not really work. 

Either (a) mark the type with a `[DisableDefaultCtor]` attribute or (b) add a test case that shows it working (that's not totally foolproof, but at least we'll know if it start failing in the future).